system/jlib/jiface.hpp

```diff
@@ -44,18 +44,16 @@ void jlib_decl raiseAssertCore(const char *assertion, const char *file, unsigned
  #define assertex(p)
  #define assert(p)
 #endif
-
 #if defined(_DEBUG)||defined(_TESTING)
 #define verifyex(p) (likely(p) ? ((void) 0) : (( (void) raiseAssertException(#p, __FILE__, __LINE__))))
 #define verify(p) (likely(p) ? ((void) 0) : (( (void) raiseAssertCore(#p, __FILE__, __LINE__))))
 #else
-#define verifyex(p) ((void) (p))
+ #define verifyex(p) ((void) (p))
```

Comment:
Well, moved from a few lines up where the space was right. I'll make the leading spaces consistent.